packages/calcite-components/src/components/popover/popover.scss

```diff
@@ -105,8 +115,7 @@
 .close-button-container {
   @apply flex overflow-hidden;
   flex: 0 0 auto;
-  border-start-end-radius: theme("borderRadius.DEFAULT");
-  border-end-end-radius: theme("borderRadius.DEFAULT");
+  --calcite-popover-border-radius: : theme("borderRadius.DEFAULT");
```

Comment:
component tokens should only be set at `:host` and this should be `--calcite-popover-corner-radius: var(--calcite-border-radius-round);`